WebView/app/src/main/java/com/android/samples/webviewdemo/MainActivity.kt

```diff
@@ -69,7 +70,7 @@ class MainActivity : AppCompatActivity() {
         //Configure asset loader with custom domain
         val assetLoader = WebViewAssetLoader.Builder()
             .setDomain("gcoleman799.github.io")
-            .addPathHandler("/Asset-Loader/", WebViewAssetLoader.AssetsPathHandler(this))
+            .addPathHandler("/Asset-Loader/assets/", WebViewAssetLoader.AssetsPathHandler(this))
```

Comment:
It can be in `"/assets/"` if we load the URL `"https://gcoleman799.github.io/assets/index.html"` as opposed to `"https://gcoleman799.github.io/Asset-Loader/assets/index.html"`. We were using the later because it is the GitHub Page that hosts the web app and the former does not exist. So we would be loading the the local web content (designed for mobile) to a different web address than where the web app lives. I am still unsure why this this can't be in `"/assets/"` when we are loading  `"https://gcoleman799.github.io/Asset-Loader/assets/index.html"`.

Reply:
> It can be in `"/assets/"` if we load the URL `"https://gcoleman799.github.io/assets/index.html"` as opposed to `"https://gcoleman799.github.io/Asset-Loader/assets/index.html"`.

Could we please do so? Your [reply on the previous PR](https://github.com/android/views-widgets-samples/pull/143#discussion_r442813688) led me to believe this was only feasible with the default domain. `"/assets/"` is the standard path so we should stick with this unless there's a reason to deviate.

> I am still unsure why this this can't be in `"/assets/"` when we are loading `"https://gcoleman799.github.io/Asset-Loader/assets/index.html"`.

The URL path (in this case, `/Asset-Loader/assets/index.html`) must match the AssetLoader's registered path (`/assets/`) + the file name (`index.html`). `/Asset-Loader/assets/index.html` != `/assets/` + `index.html`, so you should not be able to load that URL. Does this make sense?

Reply:
Yes it does, I will make a separate PR with that change

Reply:
If you're going to send a follow-up, could we revert this back to "/Asset-Loader/" to trim down this PR?

Reply:
We clarified this offline. This path is OK as-is, but could we add a comment to explain we're choosing to host our local assets as a subdirectory of the Github Pages site (which is hosted at the path "Asset-Loader")?